src/components/Illustration/enums.ts

```diff
@@ -0,0 +1,9 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+export enum IllustrationSize {
+  XXLARGE = '72',
+  XLARGE = '48',
+  LARGE = '40',
+  MEDIUM = '32',
+  SMALL = '20',
+  XSMALL = '16',
+}
```

Comment:
What's this used for?

Reply:
I don't think it is used. @swese44 I'm going to delete it for now. If you had a use in mind, let us know!